packages/spark/base/inputs/_text-input.scss

```diff
@@ -88,13 +88,10 @@ $sprk-input-huge-box-shadow: 0 3px 10px 1px rgba(0, 0, 0, 0.08) !default;
 // IE11 Override
 .sprk-b-InputContainer--huge .sprk-b-TextInputIconContainer .sprk-b-TextInput:-ms-input-placeholder,
 .sprk-b-InputContainer--huge > :first-child.sprk-b-TextInput:-ms-input-placeholder {
+  opacity: $sprk-text-input-huge-placeholder-active-opacity;
   color: transparent;
 }
-// IE11 Override
```

Comment:
We actually need this for when they want the placeholder visible and the label hidden with the added sprk-b-TextInput--label-hidden class.